aksel.nav.no/website/app/dev/produktbloggen/[...slug]/page.tsx

```diff
@@ -0,0 +1,151 @@
+import Image from "next/image";
+import { notFound } from "next/navigation";
+import { BodyLong, BodyShort, Detail, Heading } from "@navikt/ds-react";
+import { CustomPortableText } from "@/app/CustomPortableText";
+import { sanityFetch } from "@/app/_sanity/live";
+import { BLOGG_BY_SLUG_QUERY } from "@/app/_sanity/queries";
+import { urlFor } from "@/sanity/interface";
+import { abbrName, getImage } from "@/utils";
+import { parseDesignsystemSlug } from "../../(designsystemet)/slug";
+import styles from "../_ui/Produktbloggen.module.css";
+
+type Props = {
+  params: Promise<{ slug: string[] }>;
+};
+
+/* https://nextjs.org/docs/app/api-reference/file-conventions/page#props */
+export default async function Page({ params }: Props) {
+  const { slug } = await params;
+
+  const parsedSlug = parseDesignsystemSlug(slug, "produktbloggen");
+
+  const { data: page } = await sanityFetch({
+    query: BLOGG_BY_SLUG_QUERY,
+    params: { slug: parsedSlug },
+  });
+
+  const { publishedAt: publishDate } = page;
+  const authors = (page?.contributors as any)?.map((x) => x?.title) ?? [];
+
+  const imageUrl = urlFor(page?.seo?.image)
+    ?.auto("format")
+    .quality(100)
+    .url();
```

Comment:
```suggestion
  const imageUrl = urlForImage(page?.seo?.image as Image)
    .quality(100)
    .url();
```

Sanity has some type issues now, so need to "hack" it like this
```
import { Image } from "sanity";
import NextImage from "next/image";
```